test/tests/accelerations/Acceleration_By_Shooting_Method.i

```diff
@@ -616,8 +616,8 @@ dom0Scale = 25.4e-3
     mean_en = mean_en
     interp_trans_coeffs = false
     ramp_trans_coeffs = false
-    user_p_gas = 133.33
-    user_T_gas = 300
+    p_gas = 133.33
```

Comment:
Just noting that we use 133.33 here, but 133.322 in other inputs. Should discuss consistency moving forward.